packages/core/src/components/autosuggest/autosuggest.stories.ts

```diff
@@ -7,23 +7,36 @@ import readme from "./readme.md";
 
 type AutosuggestConnector = (parameters: Parameters<Parameters<typeof storiesOfAutosuggest>[1]['autosuggestDemoTemplate']>) => Parameters<typeof autosuggestTemplate>[0];
 
-const autosuggestConnector: AutosuggestConnector = ([fetchSuggestions, onSelect, onChange, onSearch, suggestOnFocus, loading, loadingLabel, notFoundLabel]) => ({
+const autosuggestConnector: AutosuggestConnector = ([fetchSuggestions, onSelect, onChange, onSearch, suggestOnFocus, loading, loadingLabel, loadingDelayed, notFoundLabel]) => ({
   suggestions: [],
   onChange: function (e) {
     onChange(e);
 
-    this.suggestions = fetchSuggestions(e.detail);
+    if (loadingDelayed) {
+      setTimeout(() => {
+        this.suggestions = fetchSuggestions(e.detail);
 
-    const suggestionsDemoPreElement = document.getElementById('suggestions-demo');
-    if (suggestionsDemoPreElement) {
-      suggestionsDemoPreElement.textContent = JSON.stringify(this.suggestions, null, 2);
+        const suggestionsDemoPreElement = document.getElementById('suggestions-demo');
+        if (suggestionsDemoPreElement) {
+          suggestionsDemoPreElement.textContent = JSON.stringify(this.suggestions, null, 2);
+        }
+      }, loadingDelayed + 1500);
+    }
+    else {
+      this.suggestions = fetchSuggestions(e.detail);
+
+      const suggestionsDemoPreElement = document.getElementById('suggestions-demo');
+      if (suggestionsDemoPreElement) {
+        suggestionsDemoPreElement.textContent = JSON.stringify(this.suggestions, null, 2);
```

Comment:
r.26 t/m r.30 herhaald zich, we kunnen dit kwijt in een `const fetchSuggestions = () => {}`